python/src/aac/plugins/gen_plugin/templates/project_readme.jinja2

```diff
@@ -16,30 +16,40 @@ Required modules are kept in the dependency sections of the pyproject.toml, and
 pip-compile command is used to add hashes to the requirements.txt file for enhanced security
 (see additional instructions below).
 
-To coincide with these changes, some changes to tox.ini and the addition of a MANIFEST.ini file were also necessary.
+To coincide with these changes, some changes to tox.ini and the addition of a MANIFEST.in file were also necessary.
 
 These lines were added to tox.ini:
+   ```
    isolated_build = True
    skipsdist = True
+   ```
 
 A MANIFEST file with these lines was added:
+   ```
    graft src
    graft tests
    include tox.ini
-   include src/rest-api/.aac
+   include src/{{ project.name }}/.aac
```

Comment:
Also still need to properly reference the definition file.  Which should be the project name by Python styled. 

Think it's something like `python_name(project.name)`